src/main/java/org/fineract/messagegateway/provider/config/ProviderConfig.java

```diff
@@ -0,0 +1,22 @@
+package org.fineract.messagegateway.provider.config;
+
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.boot.autoconfigure.condition.ConditionalOnProperty;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+
+@Configuration
+public class ProviderConfig {
+    @Value("${provider.telerivet.id}")
```

Comment:
what is this configuration? why this is not implementing any common interface?

Reply:
This is a configuration for specifying providers from application.yml.

Reply:
why the configuration name is telerivet? this is supposed to be the configuration selector ?